app/moves/controllers/dashboard.js

```diff
@@ -0,0 +1,28 @@
+const { find, reject } = require('lodash')
+
+function setTotalMoves(req, res, next) {
+  const totalMoves = {
+    label: 'moves::dashboard.filter.total',
+    filter: 'total',
+    href: find(res.locals.moveTypeNavigation, { filter: 'proposed' }).href,
+  }
+  totalMoves.value = res.locals.moveTypeNavigation.reduce(
+    (accumulator, moveType) => {
+      return (accumulator += moveType.value)
+    },
+    0
+  )
+  res.locals.moveTypeNavigation = reject(res.locals.moveTypeNavigation, {
+    filter: 'proposed',
+  })
+  res.locals.moveTypeNavigation.unshift(totalMoves)
```

Comment:
Could use ES6 spread? I find it can be a little easier to read than `unshift`:

```javascript
res.locals.moveTypeNavigation = [
  ...totalMoves,
  reject(res.locals.moveTypeNavigation, {
    filter: 'proposed',
  }),
]
```